src/C++/FieldConvertors.h

```diff
@@ -563,9 +549,7 @@ struct UtcTimeOnlyConvertor
     if( precision )
     {
       result[8] = '.';
-      if( integer_to_string_padded ( result + 9, precision, fraction )
```

Comment:
This does not need to be checked as the function integer_to_string_padded is now being tested by its own unit test in FieldConvertorsTestCase.cpp. This new unit test ensures the function does not produce the behavior being checked by this condition.